packages/driver/src/cypress/utils.js

```diff
@@ -194,7 +194,11 @@ module.exports = {
         return `Object{${len}}`
       }
 
-      return this.stringifyActualObj(value)
+      try {
+        return this.stringifyActualObj(value)
```

Comment:
What would cause this to error? Maybe we should have a test for that case.

Reply:
I noticed while debugging this issue that any circular object logged to command log will cause the test to fail because `stringifyActual` recurses infinitely. This probably isn't the handsomest fix, but it prevents the driver from failing the test due to a max call stack error. I added a test that this succeeds.